tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -870,6 +870,67 @@ def solve_tridiag(
     np.allclose(expected, out)
 
 
+def test_docstring():
+    size = 10
+    a = np_as_located_field(IDim)(np.ones((size,)))
+
+    @field_operator
+    def fieldop_with_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        return a
+
+    @program
+    def test_docstring(a: Field[[IDim], float64]) -> Field[[IDim], float64]:
+        """My docstring."""
+        fieldop_with_docstring(a, out=a)
+
+    test_docstring(a, offset_provider={})
+
+
+def test_domain(fieldview_backend):
+    size = 10
+    a = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+
+    @field_operator(backend=fieldview_backend)
+    def fieldop_domain(a: Field[[IDim, JDim], float64]) -> Field[[IDim, JDim], float64]:
+        return a + a
+
+    @program
+    def program_domain(a: Field[[IDim, JDim], float64]):
+        fieldop_domain(a, out=a, field_domain={IDim: (1, 9), JDim: (4, 6)})
+
+    program_domain(a, offset_provider={})
+
+    expected = np.asarray(a)
+    expected[1:9, 4:6] = 1 + 1
+
+    np.allclose(expected, a)
+
+
+def test_domain_tuple(fieldview_backend):
+    size = 10
+    a = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+    b = np_as_located_field(IDim, JDim)(np.ones((size, size)))
+
+    @field_operator(backend=fieldview_backend)
+    def fieldop_domain_tuple(
+        a: Field[[IDim, JDim], float64]
+    ) -> tuple[Field[[IDim, JDim], float64], Field[[IDim, JDim], float64]]:
+        return (a + a, a)
+
+    @program
+    def program_domain_tuple(a: Field[[IDim, JDim], float64], b: Field[[IDim, JDim], float64]):
+        fieldop_domain_tuple(a, out=(b, a), field_domain={IDim: (1, 9), JDim: (4, 6)})
```

Comment:
Let's add a test where start and end values are not literals but passed to the program.